test/DynamoCoreWpfTests/GraphViewTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Models;
+using Dynamo.Visualization;
+using Dynamo.Wpf.Rendering;
+using DynamoCoreWpfTests.Utility;
+
+namespace DynamoCoreWpfTests
+{
+
+    /// <summary>
+    /// This class should not be used for Dynamo View unit tests
```

Comment:
hmm good point, the reason I leave it here is because I want to make sure this class always builds fine for us to use in performance benchmarks to run. Do you have a preference?

Reply:
Yes, I would prefer just moving it to the performance project so there's less clutter in the main solution.

Reply:
OK, This will add bunch of new dependencies if you are aware. I will update this PR shortly